pkg/cmd/issue/create/create.go

```diff
@@ -285,6 +303,25 @@ func createRun(opts *CreateOptions) (err error) {
 			return
 		}
 	} else {
+		if opts.EditorMode {
+			if opts.Template != "" {
+				var template prShared.Template
+				template, err = tpl.Select(opts.Template)
+				if err != nil {
+					return
+				}
+				if tb.Title == "" {
+					tb.Title = template.Title()
+				}
+				templateNameForSubmit = template.NameForSubmit()
+				tb.Body = string(template.Body())
+			}
+
+			tb.Title, tb.Body, err = opts.TitledEditSurvey(tb.Title, tb.Body)
+			if err != nil {
+				return
+			}
```

Comment:
I assume that the later conditionals will handle empty editor content and cancel the action, correct?

Reply:
Yes.
The error handling for this part is the following.

- https://github.com/cli/cli/blob/28cc56cd456b51b3858e0556228ba26373777513/pkg/cmd/issue/create/create.go#L324-L326
    - handle editor command exit with error
- https://github.com/cli/cli/blob/28cc56cd456b51b3858e0556228ba26373777513/pkg/cmd/issue/create/create.go#L328-L331
    - handle empty title including --title flag